.github/ISSUE_TEMPLATE/give_a_talk.yml

```
@@ -0,0 +1,43 @@
+name: Give a TTW talk 
+description: Resources and checklist for giving a talk about the Turing Way
+title: "[TALK] <title>"
+labels: [talks-and-workshops, newsletter]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This template will guide you through the steps necessary to give a successful talk about The Turing Way!
+  - type: input
+    id: date
+    attributes:
+      label: Date of talk
+      description: When will you be presenting about the Turing Way?
+      placeholder: please use the format YYYY-MM-DD
+    validations:
+      required: true
+  - type: input
+    id: talk_details
+    attributes:
+      label: Details of the talk
+      description: Where are you presenting? Please provide a link to the event if available
+  - type: checkboxes
+    id: checklist
+    attributes:
+      label: Pre-talk checklist
+      description: Steps to prepare your presentation
+      options:
+        - label: Schedule your talk and let the community know in TTW Slack. If you’d like feedback and/or to schedule a practice talk, ask in the TTW Slack!
+        - label: Download template(s) from the [promotion pack](https://drive.google.com/drive/folders/1mzGmbJkPnP5q1goQesxDc_E5zAPL0eTF?usp=sharing) to ensure stylistic concistency
+        - label: If you re-used slides from a specific talk, please acknowledge the original author of the slides 
+        - label: Double check acknowledgements slide for TTW team, license info, Scriberia link, any additional acknowledgements
+        - label: Double check contact info for TTW links (book, Twitter, GitHub, Slack, newsletter), your own contact into
+  - type: checkboxes
+    id: checklist
```

```suggestion
    id: post-talk-checklist
```

This form doesn't yet load correctly because each value of `id` must be unique